mine/macros/generic_tests/null_proportion.sql

```diff
@@ -0,0 +1,21 @@
+{% macro test_null_proportion(model) %}
+
+{% set column_name = kwargs.get('column_name', kwargs.get('arg')) %}
```

Comment:
This will already be set within the context of a generic test! See [this example](https://docs.getdbt.com/docs/building-a-dbt-project/tests#generic-tests). No need for this line

Reply:
done!